komposable-architecture/src/test/java/com/toggl/komposable/reducer/CompositeReducerTests.kt

```diff
@@ -33,9 +34,9 @@ class CompositeReducerTests {
 
     @Test
     fun `reducers should be called sequentially`() = runTest {
-        combinedReducer.testReduce(originalState, TestAction.DoNothingAction) { state, effects ->
+        combinedReducer.testReduce(originalState, TestAction.DoNothingAction) { state, effect ->
             state shouldBe stateFromSecond
-            effects shouldBe effectsFromFirst + effectsFromSecond
+            effect shouldBe effectsFromFirst.merge(effectsFromSecond)
```

Comment:
Hey, merge doesn't preserve the order of the elements. It looks like the test shouldn't use the `merge` extension directly if you'd like to test the sequentiality of the returned flow.

Reply:
Yeah, this was some mindless test fix that didn't make sense. I'll remove the effects from the test for now.